README.md

```diff
@@ -33,4 +37,29 @@ Please see the [contribution guidelines](CONTRIBUTING.md).
     - Exit Survey
   - Learning & Development
     - [Training](docs/training/training-toc.md)
+- Products
+  - DreamBig
+  - OnTrack
+  - Splashkit
+- Projects
+  - T1 2022
```

Comment:
**suggestion:** Are we going to use this repo for T2 and beyond? The same company, visions, way of working, as well the same set of products? If not or unsure, I would suggest simplifying the directory structure and removing this extra layer.

Reply:
I think that this repo should be used moving forward. The whole point of the documentation is to keep building on it and to be able to have insight to past work. I put this in here so it was clear that next trimester would be added in the same folder